### PR TITLE
Aligned buttons to right

### LINKED ISSUE #6560: Make "view facility" , "View Patient" and "Notice" buttons right aligned in facility card
**Describe the bug**

"view facility" , "View Patient" and "Notice" buttons are left aligned in the facility card

**To Reproduce**

1. Go to https://care.coronasafe.in/facility
2. See error

**Expected behavior**

only move the "view facility" , "View Patient" and "Notice" buttons to the right side of the facility card

**Screenshots**

image

### PR DESCRIPTION
**Pull request by konavivekramakrishna**
### <samp>🤖 Generated by Copilot at c427646</samp>

Simplify the layout and reduce the padding of the facility card component. Remove an extra `div` element from `src/Components/Facility/FacilityCard.tsx`. The change is part of a UI improvement pull request.

## Proposed Changes

- Fixes #6560 

  Before
 
image

 After

image
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### <samp>🤖 Generated by Copilot at c427646</samp>

* Remove unnecessary `div` element and reduce vertical padding in facility card layout ([link](https://github.com/coronasafe/care_fe/pull/6573/files?diff=unified&w=0#diff-2cc6bedfa390407b524789d1828633368f746e65564574ffa86b715c469c8098L160-R260))

### CODE REVIEW

File: src/Components/Facility/FacilityCard.tsx [hunk 1]
```diff
@@ -157,109 +157,107 @@ export const FacilityCard = (props: { facility: any; userType: any }) => {
...
+                            : "text-primary-600"
+                        )}
+                      />{" "}
+                      <dt
+                        className={`my-1 text-sm font-semibold ${
                           facility.patient_count / facility.bed_count > 0.85
-                            ? "button-danger-border bg-red-500"
-                            : "button-primary-border bg-primary-100"
+                            ? "text-white"
+                            : "text-gray-700"
```
Comment (rithviknishad) [inline src/Components/Facility/FacilityCard.tsx:184]: Why was this changed? This was intentionally kept to indicate almost full utilisation of a facility.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by konavivekramakrishna** [comment]
@nihal467 
image


is it required to move the buttons to right even this view
because it not looks good
image

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
image

the grey padding for the button are reduced compared to the existing staging, was it intentional 

@rithviknishad @konavivekramakrishna

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**